### ISSUE TITLE
Regarding Initial ppts

### ISSUE THREAD

**Issue by insaaniManav** [ILUGD/volunteers-tasks#24]
@iayanpahwa @miphilomath after reading the survey results we found out that people dont like the initial ppt we present . Maybe we could have a single slide containing all the important things @nk521

**Comment by xeon-zolt** [comment]
TO-DO (discussed during volunteer call )
make a video ( take out from youtube video ) 
-  welcome slide.
-  COC 

slide for
- speaker intro.

host responsibility:
the host will play video, then introduce speaker then at the end give spear time for next meetup if speaker available give him time for introducing the next video

**Comment by xeon-zolt** [comment]
@cvam0000 @xeon-zolt will try to make to-do part 1

**Comment by miphilomath** [comment]
> @iayanpahwa @miphilomath after reading the survey results we found out that people dont like the initial ppt we present . Maybe we could have a single slide containing all the important things @nk521

Agreed. I suggest, we start the meetup without them with a basic introduction. The intro slides in itself were important for YouTube as anyone can land on any video but I think we can add that part in each video before publishing on YouTube to keep it consistent. So record a intro video, to be added in each YouTube video and introduce verbally on the meetup with the social media handles.
